sdk/servicebus/service-bus/src/util/tracing.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { OperationOptions, RestError } from "@azure/core-http";
+import { getTracer } from "@azure/core-tracing";
+import { CanonicalCode, Span, SpanKind, SpanOptions as OTSpanOptions } from "@opentelemetry/api";
+
+/**
+ * Creates a span using the global tracer.
+ * @param name The name of the operation being performed.
+ * @param operationOptions The options for the underlying http request.
+ */
+export function createSpan(
+  operationName: string,
+  operationOptions: OperationOptions = {}
+): { span: Span; updatedOperationOptions: OperationOptions } {
+  const tracer = getTracer();
+  const spanOptions: OTSpanOptions = {
+    ...operationOptions.tracingOptions?.spanOptions,
+    kind: SpanKind.INTERNAL
+  };
+
+  const span = tracer.startSpan(`Azure.ServiceBus.${operationName}`, spanOptions);
+  span.setAttribute("az.namespace", "Microsoft.ServiceBus");
+
+  let newOptions = operationOptions.tracingOptions?.spanOptions || {};
```

Comment:
Just because there are so many different options we are working with, can we rename `newOptions` here to `newSpanOptions` or `updatedSpanOptions`?

Reply:
Updated to `newSpanOptions`